viite-UI/src/view/RoadAddressProjectEditForm.js

```diff
@@ -204,9 +205,17 @@
     };
 
     var changeDropDownValue = function (statusCode) {
+      var rootElement = $('#feature-attributes');
       if (statusCode === STATUS_UNCHANGED) {
         $("#dropDown").val('ennallaan').change();
       }
+      else if(statusCode === STATUS_NEW_LINK){
+        $("#dropDown").val('uusi').change();
```

Comment:
We should work away the Finnish constant strings from the code and replace them with "static" variables or other kind of references.